README.md

```diff
@@ -30,6 +30,9 @@ To get a local copy up and running follow these simple example steps.
 `git clone https://github.com/tiagorahal/blog-app.git`
 - CD into the cloned directory
 - Run `bundle install`
+- Run `npm i`
+- Run `rails db;create`
```

Comment:
- [ ] [OPTIONAL] There's a little error here, this should be `rails db:create`, please fix this.